src/internal.c

```diff
@@ -3869,7 +3869,7 @@ static int DoGlobalRequest(WOLFSSH* ssh,
         *idx += len;
 
         if (wantReply)
-            ret = SendRequestFailure(ssh);
+            ret = SendRequestSuccess(ssh, ~WS_SUCCESS);
```

Comment:
If the second parameter is treated as boolean, ~WS_SUCCESS can just be 0. WS_SUCCESS is supposed to be a named return code, not treated as a number.

Reply:
Done